### PR TITLE
DOC: Update old Google Code and SourceForge links

### PR DESCRIPTION
**Pull request by proinsias**

### CODE REVIEW

File: pandas/io/auth.py [hunk 1]
```diff
@@ -30,7 +30,7 @@ class AuthenticationConfigError(ValueError):
 
    %s
 
-with information from the APIs Console <https://code.google.com/apis/console>.
+with information from the APIs Console <https://console.developers.google.com/iam-admin/projects>.
```
Comment (jorisvandenbossche) [inline pandas/io/auth.py]: This gives a lint error (line to long), that's why travis is failing. Can you put the link on a new line?

**Comment by jorisvandenbossche** [comment]
@proinsias  Thanks a lot!

Only a small PEP8 issue

File: README.md [hunk 2]
```diff
@@ -170,7 +170,7 @@ conda install pandas
 - [SciPy](http://www.scipy.org): miscellaneous statistical functions
 - [PyTables](http://www.pytables.org): necessary for HDF5-based storage
 - [SQLAlchemy](http://www.sqlalchemy.org): for SQL database support. Version 0.8.1 or higher recommended.
-- [matplotlib](http://matplotlib.sourceforge.net/): for plotting
+- [matplotlib](http://matplotlib.org/): for plotting
 - [statsmodels](http://statsmodels.sourceforge.net/)
```
Comment (TomAugspurger) [inline README.md]: Can you also update this to http://www.statsmodels.org/stable/index.html? Thanks.

**Comment by proinsias** [comment]
Updated all statsmodels links, and double-checked pep8!

**Comment by jorisvandenbossche** [comment]
Thanks a lot!

**Pull request merged**